### PR TITLE
#3143 fixes bug in (swing) VLocator grid table editor

### PR DESCRIPTION
**Pull request by homebeaver**
Hi @yamelsenih - patch for bug #3143

pls review & merge 
regards

### CODE REVIEW

**Comment by yamelsenih** [comment]
Hi @homebeaver I will test it

File: client/src/org/compiere/grid/ed/VLocator.java [hunk 1]
```diff
@@ -625,7 +625,10 @@ private void setDefault_Locator_ID()
 			return;
 		}
 		MWarehouse wh = MWarehouse.get(Env.getCtx(), M_Warehouse_ID);
-		if (wh == null || wh.get_ID() <= 0) {
```
Comment (yamelsenih) [inline client/src/org/compiere/grid/ed/VLocator.java:628]: Have you a similar change for WLocator? Please this change should be applied also for ZK UI

Comment (homebeaver) [reply client/src/org/compiere/grid/ed/VLocator.java:628]: Hi @yamelsenih,
No. The problem targetes only to swing.

**Comment by e-Evolution** [comment]
@homebeaver , I think that same problem that show warehouse deactivated in ZK is wrong 

https://github.com/adempiere/adempiere/blob/f06f4245714ba4a3da72639e5492f14e06c46888/zkwebui/WEB-INF/src/org/adempiere/webui/window/WLocatorDialog.java#L396

Can adding the code functionality to not show warehouse inactive, please ?

**Comment by marcalwestf** [comment]
@homebeaver  hi Eugen, it seems that the PR is OK, only the same has to be applied to ZK.
I hope the corresponding change in _WLocator_ is not too difficult.
Thanks!

**Comment by homebeaver** [comment]
> 
> 
> ... only the same has to be applied to ZK.
> I hope the corresponding change in _WLocator_ is not too difficult.

Hi @marcalwestf - I've checked it once more. 
And as I wrote on 14 Jul 2020 for me _the problem targetes only to swing_.

regards

P.S. teo_sarca added the FR [ 1661546 ] `setDefault_Locator_ID()` on 2007-02-20 and only to swing class `grid.ed.VLocator`.

**Comment by yamelsenih** [comment]
Hi @homebeaver can you change the validation named by @e-Evolution ? Just it for merge

**Comment by yamelsenih** [comment]
Hi @homebeaver how are you?

I add a pull request for add the @e-Evolution comment here: https://github.com/klst-de/adempiere/pull/1

Please just apporove this in your repo for merge this PR


Best regards

**Comment by yamelsenih** [comment]
Thanks @homebeaver

**Pull request merged**